### DISCUSSION TITLE
В результаты поиска попадает php код из записи

### DISCUSSION THREAD

**Discussion by nvda-ru** [maxsite/cms#478] [Q&A]
Доброго времени суток.
Всех с праздниками: Наступившим новым 2022 годом, а также с наступающим Рождеством.

Недавно обнаружил что в результаты поиска, попадает, скажем так, технический код.
То есть, код php вставленный в записи, между тегами [php] ... [/php], шорткоды [cut] и ушек [ushka=name].
Так понимаю, что можно случайно найти теги бибикода или псевдокода, всё то что в квадратных скобках.

А можно ли, каким-то образом всё это исключить из результатов поиска.
Например, через список исключений. В него вручную заносить всё то что нужно игнорировать при поиске напрямую или при выдаче результатов. Или автоматически игнорировать всё то что в квадратных скобках.
Например, задал в поиск одно слово, но это слово было близко к указанным тегам, поэтому в результатах было показано это слово и рядом фрагменты php кода и ли ушки, теги псевдокода и тд.

У меня на сайте, незрячие пользователи используют поиск по сайту, и не очень хорошо, если в результатах они будут находить все эти технические коды и теги, да и чтобы в прямую их не находили, если например напишут запрос "include".
Если это слово между тегами [php] include [/php], то чтобы этот текст игнорировался.

Может для администратора это удобно, быстро найти в записях какое-то вхождение php, но пользователям это уж точно не нужно.

**Comment by maxsite** [comment]
Поиск сделал через вхождение LIKE в MySQL. Там нет возможности как-то ограничивать фразы. Ищется ровно то, что есть в базе. В качестве альтернативы можно подключить поиск гугла. Других вариантов я не знаю.
